packages/mds-agency/tests/db-integration-tests.ts

```diff
@@ -41,6 +41,7 @@ import {
 import db from '@mds-core/mds-db'
 import cache from '@mds-core/mds-cache'
 import stream from '@mds-core/mds-stream'
+import { shutdown as socketShutdown } from '@mds-core/mds-web-sockets'
```

Comment:
Why not `import socket from '@mds-core/mds-web-sockets'` for the sake of consistency?